server/lib/gateway/gateway.init.js

```diff
@@ -44,13 +44,12 @@ async function init() {
   // schedule backup at midnight
   const timezone = await this.variable.getValue(SYSTEM_VARIABLE_NAMES.TIMEZONE);
 
-  const rule = new this.schedule.RecurrenceRule();
-  rule.tz = timezone;
-  rule.hour = 0;
-  rule.minute = 0;
-  rule.second = 0;
-
-  this.backupSchedule = this.schedule.scheduleJob(rule, this.checkIfBackupNeeded.bind(this));
+  const rule = { tz: timezone, hour: 0, minute: 0, second: 0 };
+  this.backupSchedule = this.scheduler.scheduleJob(
+    JOB_TYPES.GLADYS_GATEWAY_BACKUP,
```

Comment:
Why do you use the JOB_TYPES here? I think we are expecting to emit an event here, so the proper object should be EVENTS, JOB_TYPES is only the key for job types

Reply:
According to `node-schedule` examples, I can ignore the name of jobs.
This means that we have to keep generated job in our process to cancel or reschedule it as a singleton if needed.